src/appshell/qml/NotationPage/NotationPage.qml

```diff
@@ -91,6 +91,8 @@ DockPage {
     }
 
     readonly property int verticalPanelDefaultWidth: 300
+    readonly property int verticalPanelMinimumHeight: 200
```

Comment:
I think we should have different minimum values for floating and docked panels. This value may be too big for vertically stacked panels. For example, the status bar disappears if I place the panels vertically on my screen (because there is not enough free vertical space for it)

<img width="304" alt="Screenshot 2022-01-10 at 11 06 34" src="https://user-images.githubusercontent.com/67867444/148741439-11658cb4-7481-4446-8bc6-0760957eff4d.png">

Reply:
@bkunda could you advise us on the most appropriate min/max sizes for vertical/horizontal panels, depending on whether they are docked or not?

Reply:
Good point. I've added another commit to allow this. I've gone for min height docked of 50px and min width docked of 100px, but can easily change if advised.